src/us/kbase/workspace/WorkspaceServer.java

```diff
@@ -121,7 +121,7 @@
 	private final Types types;
 	private final WorkspaceAdministration wsadmin;
 	
-	private final URL handleManagerUrl;
+	private final URL handleServiceUrl;
```

Comment:
I think the handle service url can be deleted as it's stored in WorkspaceServerMethods and the checking method is called there. AFAICT the check method below is now unused.

Reply:
👍